IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -355,6 +356,19 @@ - (void)decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction
         return;
     }
     
+    // Handle anchor links that were clicked
+    if ([navigationAction navigationType] == WKNavigationTypeLinkActivated)
+    {
+        //Open secure web links with target=new window in default browser or non-web links with URL schemes that can be opened by the application
+        if (([requestURL.scheme.lowercaseString isEqualToString:@"https"] && !navigationAction.targetFrame.isMainFrame) || ![requestURL.scheme.lowercaseString hasPrefix:@"http"])
+        {
+            MSID_LOG_INFO(self.context, @"Opening URL outside embedded webview : %@",requestURLString);
```

Comment:
Is it safe to print the request url out? Do we encrypt it in logs?

Reply:
Good catch, Likely they won't contain PII but I will replace it with MSID_LOG_INFO_PII in case.

Reply:
Addressed

Reply:
good catch, we shouldn't print URL even in PII logs as it might contain secrets.

Reply:
@oldalton Is it okay to log the scheme and the host? If not, I'll change the log to just say "Opening URL outside embedded webview"

Reply:
yes, host and scheme are OK

Reply:
Addressed: Logging scheme and host